support/refc/runtime.c

```diff
@@ -1,5 +1,13 @@
 #include "runtime.h"
 
+#ifdef INTEGER_USE_LIBBF
+extern void init_bf();
+#endif
+
+void init_idris2() {
+  init_bf();
```

Comment:
This should have some C preprocessor stuff around it, shouldn't it? I believe that's why the RefC tests are all failing

Reply:
You're right. I was in a hurry when doing some clean-ups.